src/main/java/core/basesyntax/util/PropertyReader.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.util;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertyReader {
+    private static final Properties properties = new Properties();
+
+    static {
+        try (InputStream inputStream = PropertyReader.class
+                .getClassLoader()
+                .getResourceAsStream("application.properties")) {
+            if (inputStream != null) {
+                properties.load(inputStream);
+            } else {
+                throw new RuntimeException("Sorry, unable to find application.properties");
+            }
+        } catch (IOException e) {
```

Comment:
According to the checklist, don't ignore exceptions in try-catch blocks. It is recommended to rethrow a RuntimeException with an informative message and an exception object. Here, you've done it correctly.

Reply:
the logic was changed